### ISSUE TITLE
Get/Create/Join spaces by user account address

### ISSUE THREAD

**Issue by nischitpra** [snapshot-labs/snapshot.js#562]
**Is your feature request related to a problem? Please describe.**
1. I need to search list of spaces by user account address
2. I need to create and join spaces using `snapshot.js` but the documentation only has a placeholder `...`. When will the api be ready? 

**Describe the solution you'd like**
`await client.spaces.get("client account address")`
`await client.spaces.create("awesome.eth", "client account address")`
`await client.spaces.join("awesome.eth", "client account address")`

**Additional context**
The graphql query for multiple spaces doesn't allow searching by members. Can add there as well.

**Comment by ChaituVR** [comment]
Yes we should document how to follow and unfollow spaces using snapshot.js

**Comment by HiperMaximus** [comment]
bump to document this
